pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -49,6 +49,8 @@ const (
 
 	portNameMetrics       = "metrics"
 	portMetrics     int32 = 8085
+
+	clusterAutoScalerPriorityConfigMapName = "cluster-autoscaler-priority-expander"
```

Comment:
Since this name is only used at a single place, I would remove this const and inline the name where needed.